### ISSUE TITLE
to stop the live capture

### ISSUE THREAD

**Issue by hmchaitra** [KimiNewt/pyshark#233]
hi,
I am trying to define two different function to start a capture and stop the capture,inside stop capture function i have written os.system(kill tshark),but its not working how to stop the livecapture

**Comment by taseppa** [comment]
I'm kind of wondering the same. It seems a new tshark process is created for each capture and even when the capture is done the process still exists.

**Comment by hmchaitra** [comment]
try , pkill -9 tshark.

**Comment by taseppa** [comment]
I use this command when im done with the program and want to clear the tshark processes. There must however a way to clear the capture within the code while the application is running.

**Comment by hmchaitra** [comment]
okie ok..after time interval specified in sniff function, captures stops, but the process still exists.

**Comment by Tsajan** [comment]
I still don't know how to resolve this issue! Can someone guide me here how to stop sniffing of livecapture?

Thanks

**Comment by MxLinux** [comment]
I've looked through the capture source, and don't see anything that actually returns the PID of a newly spawned tshark instance. Is this implemented? It looks like (though I may be mistaken...) _get_tshark_process returns a Process object but A) spawns the process when called and B) even when calling capture._get_tshark_process.pid() (Process should have attribute pid() per documentation of subprocess library), I'm receiving an error. 

I'm creating a small, simple interface to monitor traffic to/from specific IPs and would like to use a simple text menu to add/remove IPs from this. The goal is to, upon creation, obtain the PID so that it can be selectively killed (ideally with children) upon deletion. I would rather not kill all instances of tshark/tcpdump so that when a single IP is removed, the rest remain in capture. 

TL;DR If someone could point me in the direction of how to assign PID of tshark process to a variable when starting the capture, that would be immensely helpful.

**Comment by TobiasOberle** [comment]
No furthercoming here? I also need a solution to this.

**Comment by nagaditya39** [comment]
hi, how was this completed? there hasn't been a solution outlined
